components/callbacks/executors.go

```diff
@@ -147,8 +149,10 @@ func (e taskExecutor) executeInvocationTask(
 	err = e.saveResult(ctx, env, ref, response, callErr)
 
 	if callErr != nil {
+		e.Logger.Error("Callback request failed", tag.Error(callErr))
 		err = queues.NewDestinationDownError(callErr.Error(), err)
 	} else if isRetryableHTTPResponse(response) {
+		e.Logger.Error("Callback request failed", tag.Cause(response.Status))
```

Comment:
[Nit] can we also have the error message distinguish between retryable vs non-retryable rather than having to rely on error vs cause tag